header/interface.go

```diff
@@ -125,6 +125,9 @@ type Getter interface {
 
 	// GetRangeByHeight returns the given range [from:to) of ExtendedHeaders.
 	GetRangeByHeight(ctx context.Context, from, to uint64) ([]*ExtendedHeader, error)
+
+	// GetVerifiedRange returns verified range from the provided ExtendedHeader to the provided height.
+	GetVerifiedRange(context.Context, *ExtendedHeader, uint64) ([]*ExtendedHeader, error)
```

Comment:
Just for consistency with GetRangeByHeight which keeps param names so they are referenceable from the comment 
```suggestion
	GetVerifiedRange(ctx context.Context, from *ExtendedHeader, to uint64) ([]*ExtendedHeader, error)
```